src/btree/col_srch.c

```diff
@@ -73,10 +73,16 @@ __wt_col_search(
     uint64_t recno;
     uint32_t base, indx, limit, read_flags;
     int depth;
+    /*
+     * In FLCS, when looking for an implicit record, we want to return it as long as a greater
```

Comment:
I feel the comment is misleading. We will not look for implicit records, but the search needs to return them if there is no valid record. How about something like the one below?

`As long as there is a greater record in the tree, FLCS should return the implicit record that it is searching for.`

Reply:
Thank you for the suggestion, see 99b0e3e6675b30bf5aaf257894825162904b0d6e.